android/src/main/java/com/mattermost/networkclient/interceptors/ExponentialRetryInterceptor.kt

```diff
@@ -17,6 +17,6 @@ class ExponentialRetryInterceptor(
     }
 
     override fun getWaitInterval(attempts: Int): Long {
-        return (exponentialBackoffBase.pow(attempts) * exponentialBackoffScale).toLong()
+        return (exponentialBackoffBase.pow(attempts) * exponentialBackoffScale * 1000).toLong()
```

Comment:
@migbot Should we do the same on iOS?

Reply:
Or to make it consistent with linear retry, exponentialBackoffBase and exponentialBackoffScale fields in example app to be settable in millis?